src/assets/styles/modules/image-with-text.scss

```diff
@@ -0,0 +1,8 @@
+.image-with-text {
+  display: flex;
```

Comment:
Do we care that this doesn't look great on mobile? I can't decide whether we should specify these styles for `$medium-up` only. Thoughts @tauthomas01 @t-kelly? Also, this same question applies to future CSS we'll be adding to our required sections.

If we start specifying layout differences in CSS, the logic for the image alignment setting won't apply to mobile (it wouldn't be left/right, but rather top/bottom) and we'd probably have to add a label for that setting. I'm leaning towards keeping this as is.

Reply:
Personally, I also tend to keep as it is. You are right about the label name as it will lose its meaning if we introduce responsive layouts. I think if we start adding breakpoints to all standard sections, I'm afraid people will start complaining on their devices (e.g. Windows tablet that act like desktop, iPhone 10 on landscape is broken, etc.). Maybe we can add a warning message in our documentation that states "the visual layout might varies as we added minimal aesthetic" or something similar.

@t-kelly : what do you think?

Reply:
Yeah totally, let's forget responsiveness and keep it barebones.

Reply:
What if we set a width for the image, and when there isn't enough space, the text and image get stacked? Minimal solution?